package.json

```diff
@@ -30,10 +30,11 @@
     "strip-ansi-cli": "^2.0.0"
   },
   "dependencies": {
+    "assemblyscript": "^0.8.0",
     "bs58": "^4.0.1",
     "jest-environment-node": "^24.5.0",
     "ncp": "^2.0.0",
-    "near-assemblyscript": "^0.7.4",
+    "near-bindgen-as": "github:nearprotocol/near-bindgen-as#v1.1.0",
```

Comment:
Let's not depend on GitHub version anymore. Just publish it on NPM. Should probably be `0.x.z` version still before main net though.

Reply:
Yeah I was going to reach out about setting up a github action to publish near-bindgen-as.

Reply:
I actually prefer explicit manual publish using https://github.com/sindresorhus/np